library/src/main/java/com/airbnb/android/airmapview/RuntimePermissionUtils.java

```diff
@@ -0,0 +1,99 @@
+package com.airbnb.android.airmapview;
+
+import android.annotation.TargetApi;
+import android.app.Activity;
+import android.content.Context;
+import android.content.pm.PackageManager;
+import android.support.v4.app.ActivityCompat;
+import android.support.v4.app.Fragment;
+
+import static android.support.v4.content.PermissionChecker.checkSelfPermission;
+
+/**
+ * Utility class that handles Runtime location permissions for Android M
```

Comment:
remove the "for Android M" from this comment as it will be outdated once a new version is released